src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -24,7 +24,7 @@ use clap::Subcommand;
 use dialoguer::{theme::ColorfulTheme, Confirm};
 use futures::TryStreamExt;
 use fvm_ipld_blockstore::Blockstore;
-use fvm_shared3::address::Network;
+use fvm_shared4::address::Network;
```

Comment:
Perhaps we could export `latest` from this shim to have this versioning mostly contained there.

Reply:
Fixed.